staging/jenkins/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+name: jenkins
+home: https://jenkins.io/
+version: 1.7.3
+appVersion: lts
+description: Open source continuous integration server. It supports multiple SCM tools
```

Comment:
This should probably say this is D2IQ's version of Jenkins which has been adapted to work on Konvoy?

Reply:
:heavy_plus_sign::heavy_plus_sign: